state-of-our-networks.md

```diff
@@ -34,19 +34,19 @@ order: 0
 
 ***
 
-Brought to you by Toronto Mesh, '**State of our Networks**' is a one-day, hands-on conference about the past, present, and future of building our own network infrastructures to communicate and access the internet in our city.
+Brought to you by Toronto Mesh, '**State of our Networks**' is a one-day, hands-on conference about the past, present, and future of building our own network infrastructures to communicate and access the internet in our city. Join us for:
 
-On {{ page.date | date: '%B %d, %Y' }} from {{page.startTime | date:"%-I:%M%P "}} to {{page.endTime | date:"%-I:%M%P "}} join us for a one-day hands-on conference to:
+- hands-on workshops* to build mesh nodes and learn about low-power networks
+- talks about a history of Toronto's internet
+- planning how to build resilient and accessible communications networks and connections to the internet
 
-- learn and participate in hands on workshops led by current networking projects in the city of Toronto
-- hear about the history of community networks in our city
-- discuss how we can build a resilient network that addresses barriers in the future
+Please see our [schedule below](#schedule) for more details.
 
-Toronto Mesh welcomes people from all backgrounds and technical skill levels and aims to provide a harassment-free environment. This event has a [Code of Conduct]({{ site.baseurl }}/code-of-conduct/).
+*_Note: Workshops require a laptop (any operating system), but attendees are also welcome to bring [Raspberry Pis](https://en.wikipedia.org/wiki/Raspberry_Pi) or other [single-board computers](https://en.wikipedia.org/wiki/Single-board_computer) pre-flashed with Linux._
```

Comment:
I was thinking if people jumped right to the schedule it would be nice to have the reminder again? I think it should be clear which the laptop sessions are is all. The copy doesn't have to be the same though